charts/oidc-proxy/values.yaml

```diff
@@ -56,6 +56,7 @@ service:
 
 ingress:
   enabled: true
+  sslPolicy:
```

Comment:
Do we not need to give it an empty string or object?

Reply:
Ah, I see, with the if statement, probably not

Reply:
it's mostly for demonstrating what values can be over written? I've been lazy and haven't written a schema for any of the datarepo charts yet
https://helm.sh/docs/helm/helm_show_values/